foca/foca.py

```diff
@@ -0,0 +1,97 @@
+""" Definition of an entry-point function for setting up and initializing a
+FOCA based microservice."""
+
+from foca.api.register_openapi import register_openapi
+from foca.config.log_config import configure_logging
+from foca.database.register_mongodb import register_mongodb
+from foca.errors.errors import register_error_handlers
+from foca.factories.connexion_app import create_connexion_app
+from foca.factories.celery_app import create_celery_app
+from foca.security.cors import enable_cors
+from foca.config.config_handler import Config
+from connexion import App
+from typing import Mapping, Any
+
+
+def foca(config: Mapping[str, Any]
+         ) -> App:
+    """ Initialization of a FOCA based microservice.
+
+    Args:
+        config: A Dict object, parsed from a proper YAML configuration file,

```

Comment:
You can just refer to the model class that @kushagra189 is writing for the format here, as that is the ultimate source of truth. Otherwise we have to always remember that to make changes in the comment here when there are changes in the config model (and that's definitely not gonna happen). People can then look it up in the API docs and find all they need to know.

Reply:
However, copy this docstring as a comment to the #foca channel or the config issue #43 so that we will be able to make use of it when writing the docstring of the config handler.